### ISSUE TITLE
Analyse some real-world projects in CI

### ISSUE THREAD

**Issue by vemv** [clojure-emacs/refactor-nrepl#322]
One thing that has helped me while developing Eastwood (another t.ana-based tool) is having third-party projects as submodules:

https://github.com/jonase/eastwood/tree/29eb37a56055c304c6222f43cb6de1c25192358f/.circleci

This way one can assert in CI that they can be in fact analysed without errors.

**Comment by bbatsov** [comment]
Yeah, that's always a great idea for code analyzers. While I don't do in RuboCop's CI (it'd be crazy slow), we do run RuboCop from time to time on a large set of OSS Ruby projects.

**Comment by bbatsov** [comment]
Btw, I had no idea you're maintaining Eastwood as well these days - you truly are on fire! 🚀 🔥 🥇

**Comment by vemv** [comment]
> Btw, I had no idea you're maintaining Eastwood as well these days

🍻 it's been a motivating adventure to keep the old-school tools alive and kicking. I really appreciate our lispy, runtimey approach

**Comment by bbatsov** [comment]
You and me both. For me that's the very essense of Lisp programming.
